multiqc/multiqc.py

```diff
@@ -746,6 +746,27 @@ def __rich_measure__(self, console: rich.console.Console, options: rich.console.
         report.runtimes["mods"][run_module_names[mod_idx]] = time.time() - mod_starttime
     report.runtimes["total_mods"] = time.time() - total_mods_starttime
 
+    # Parse software version from config if provided
+    versions_from_config = software_versions.load_versions_from_config(config)
+    if versions_from_config:
+        for software, versions in versions_from_config.items():
+            # Try to find if the software is listed among the executed modules.
+            # Unlisted software are still reported in the `Software Versions` section.
+            versions = list(set(map(str, versions)))
+            versions.sort(reverse=True)
```

Comment:
I think this fits better in the `load_versions_from_config` function, along with the other cleanup we have there..